caddy/caddymain/run.go

```diff
@@ -126,6 +126,9 @@ func Run() {
 		mustLogFatalf("%v", err)
 	}
 
+	// Executes Instance Startup events
+	caddy.EmitEvent(caddy.InstanceStartupEvent, nil)
```

Comment:
What should we pass as data to this event?

Reply:
instance